### ISSUE TITLE
Typo in documentation

### ISSUE THREAD

**Issue by gabrielhdt** [Deducteam/Dedukti#255]
The `README.md` states that injectivity is enabled with `inj` (https://github.com/Deducteam/Dedukti#injectivity), but it is enabled with `injectivity`

**Comment by fblanqui** [comment]
It could be useful to include https://github.com/Deducteam/lambdapi/blob/master/docs/syntax_dedukti.bnf in Dedukti (this grammar is generated in https://github.com/Deducteam/lambdapi/blob/master/docs/Makefile).

**Comment by gabrielhdt** [comment]
It is already at the root of the repository https://github.com/Deducteam/Dedukti/blob/master/syntax.bnf

**Comment by gabrielhdt** [comment]
It looks out of date, there is no mention of `injective`.

**Comment by fblanqui** [comment]
But is this file automatically generated from the grammar?

**Comment by gabrielhdt** [comment]
Yes it is https://github.com/Deducteam/Dedukti/blob/c65e7e66fe4bb8285db1856b8fe2a2532da2bd50/Makefile#L165

**Comment by fblanqui** [comment]
Fixed in https://github.com/Deducteam/Dedukti/commit/d3bd1f26c0e28157f60ec5145caf4f86c5c1dd4b
